IntegrationAdapters/wwwroot/index.html

```diff
@@ -0,0 +1,31 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+    <style>
+        .background {
+            background-image: url('images/background.jpg');
+            background-position:50% 50%;
+            background-attachment: fixed;
+            background-repeat: no-repeat;
+            background-size: cover;
+        }
+        .registrationButton 
+            {
+                position: absolute;
+                top: 10px;
+                right: 10px;
+            }
+    </style>

```

Comment:
This could also be in a separate css file, also you don't need to use the same file for all pages